programs/eosio-launcher/main.cpp

```diff
@@ -555,6 +560,10 @@ launcher_def::initialize (const variables_map &vmap) {
     }
   }
 
+  if (vmap.count("cluster-id")) {
+    cluster_id = vmap["cluster-id"].as<uint32_t>();
```

Comment:
`cluster_id` could be a `uint16_t` and you could assert that it is less than `max_num_clusters` here.

Reply:
Latest changes switch to using `uint16_t`. But I would still like to see an assertion that `cluster_id <= port_constants::max_num_clusters` to provide a helpful error message early on if the user provides a cluster ID that is too large.